airiam/find_unused/PolicyAnalyzer.py

```diff
@@ -56,6 +56,11 @@ def policy_is_write_access(policy_document):
                                           None if action_map.get(action_service,None) is None else action_map[action_service]['privileges']))
             except StopIteration:
                 action_objs = []
+                print(action)
+                print(action_map)
+                print(action_service)
+                print(action_map[action_service]['privileges'])
+                print(action_map.get(action_service,None))
```

Comment:
Can we make this log a little more explicit? i.e.:
1. Use `logging.warning` / `logging.error` and specify this `action` & `action_service` was not found.
2. Lower the logging level of the `action_map` abd tge `privileges` to `logging.debug`
?